include/superior_mysqlpp/config.hpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * Author: Jiří Kozlovský
+ */
+
+#pragma once
+
+#include <string>
+
+namespace SuperiorMySqlpp
+{
+    struct SslConfiguration
+    {
+        const char* keyPath = nullptr;
+        const char* certificatePath = nullptr;
+        const char* certificationAuthorityPath = nullptr;
+        const char* trustedCertificateDirPath = nullptr;
+        const char* allowableCiphers = nullptr;
+    };
+
+    class ConnectionConfig {
```

Comment:
Please rename this to ConnectionConfiguration for consistency with SslConfiguration